bgpd/bgp_clist.h

```diff
@@ -109,7 +109,6 @@ struct community_list_handler {
 };
 
 /* Error code of community-list.  */
-#define COMMUNITY_LIST_ERR_CANT_FIND_LIST        -1
 #define COMMUNITY_LIST_ERR_MALFORMED_VAL         -2
```

Comment:
Also, can you enumerate all the codes below? Or write a comment why it's -2 (the first one).

Reply:
fixed it